CommonLibSF/include/SFSE/Utilities.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "REL/Relocation.h"
+#include "SFSE/API.h"
+
+namespace SFSE
+{
+	namespace stl
+	{
+		template <class T, unsigned TSize = 5>
+		constexpr void write_thunk_call() noexcept
+		{
+			SFSE::AllocTrampoline(14);
+			auto& trampoline{ SFSE::GetTrampoline() };
```

Comment:
Prefer using ` auto& trampoline = SFSE::GetTrampoline();` here

Reply:
Why? I generally use value-initialization and this syntax works fine

Reply:
Because we have been using these conventions in the past CLib, and since we kept using them here, let's try not to bring personal coding style in.

Reply:
What? Past CommonLibs use value-initialization all over the place too, dunno what convention you're talking about

Reply:
Searching for `auto& trampoline{ SKSE::GetTrampoline() };` gives us 4 hits, all of which are from your projects, vs 304 results

Reply:
Show me where we initialized a reference with value initialization in past CLib.

Reply:
What does number of hits have to do with anything? I would argue it's completely arbitrary that `=` was used here and `{}` works just as well. If we're concerned about search hits for individual lines of code then we should probably ensure that everything else looks identical too 🤨 

Marking resolved since the requested change was already made